src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php

```diff
@@ -100,16 +106,30 @@ private function getIsGrantedSubject(string|Expression $subjectRef, Request $req
 
     private function getIsGrantedString(IsGranted $isGranted): string
     {
-        $processValue = fn ($value) => \sprintf($value instanceof Expression ? 'new Expression("%s")' : '"%s"', $value);
+        $processValue = function ($value) {
+            if ($value instanceof Expression) {
+                return \sprintf('new Expression("%s")', $value);
+            }
+
+            if ($value instanceof \Closure) {
+                return 'callable';
```

Comment:
I suggest using `\Closure`, or maybe getting the actual name of the closure as generated by PHP in error messages maybe.

Reply:
Tried using the closure name as returned by the reflection API (maybe there's another way to do it?).

You can see how it looks like [here](https://github.com/symfony/symfony/pull/59150/files#diff-a7ab5aab5c5337af307341aaae9338a6893955b981d018018b79d3849f368e5eR248). It's pretty verbose, bit the accuracy of the output worth it to me.